code/modules/mob/living/simple_animal/abnormality/teth/my_sweet_home.dm

```diff
@@ -0,0 +1,99 @@
+//Brain go brrrr.
+/mob/living/simple_animal/hostile/abnormality/my_sweet_home
+    name = "My Sweet Home"
+    desc = "This cozy little house is a safe nest built only for you. Everything is here for you..."
+    icon = 'ModularTegustation/Teguicons/96x64.dmi'
+    icon_state = "sweet_home"
+    icon_living = "sweet_home"
+    icon_dead = "sweet_home_death"
+
+    threat_level = TETH_LEVEL
+    can_breach = TRUE
+    del_on_death = FALSE
+    maxHealth = 600
+    health = 500
+    move_to_delay = 4
+    damage_coeff = list(BRUTE = 1, RED_DAMAGE = 0.9, WHITE_DAMAGE = 0.9, BLACK_DAMAGE = 1.2, PALE_DAMAGE = 2)
+    melee_damage_lower = 6
+    melee_damage_upper = 12
+    melee_damage_type = RED_DAMAGE
+    armortype = RED_DAMAGE
+    melee_queue_distance = 1
+    retreat_distance = 0
```

Comment:
0?

Reply:
Makes the abno actually move instead of standing still and face tanking.